tests/unit/test_charm.py

```diff
@@ -915,3 +915,214 @@ def test_postgres_patch_storage_config_defaults_to_database_relation(self):
         }
         environment = plan.to_dict()["services"]["livepatch"]["environment"]
         self.assertEqual(environment, environment | required_environment)
+
+    def test_pro_airgapped_server_relation__success(self):
```

Comment:
Can we put these tests in a separate test class? I think it's time to split this test class up so that when things break it's easier to identify what's going wrong.

Reply:
I agree with this suggestion, but can we do it in another PR?

Reply:
Yes for sure